### ISSUE TITLE
[Bug]: test

### ISSUE THREAD

**Issue by joeschmoey** [jrlegrand/issue-template#2]
### Contact Details

joeschmoey@yahoo.com

### What happened?

A bug happened! test

### Version

1.0.2 (Default)

### What browsers are you seeing the problem on?

Firefox

### Relevant log output

```shell
output
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by jrlegrand** [comment]
I’m replying to this Sent from my iPhoneOn Jun 29, 2024, at 9:55 PM, joeschmoey ***@***.***> wrote:﻿
Contact Details
***@***.***
What happened?
A bug happened! test
Version
1.0.2 (Default)
What browsers are you seeing the problem on?
Firefox
Relevant log output
output
Code of Conduct

 I agree to follow this project's Code of Conduct

**Comment by joeschmoey** [comment]
Yeah that’s not the issue.  Sent from my iPhoneOn Jun 29, 2024, at 10:07 PM, Joseph LeGrand ***@***.***> wrote:﻿
I’m replying to this Sent from my iPhoneOn Jun 29, 2024, at 9:55 PM, joeschmoey ***@***.***> wrote:﻿

Contact Details

***@***.***

What happened?

A bug happened! test

Version

1.0.2 (Default)

What browsers are you seeing the problem on?

Firefox

Relevant log output

output

Code of Conduct



 I agree to follow this project's Code of Conduct

**Comment by joeschmoey** [comment]
Something something
